patitasuy/src/app/components/edit-profile/edit-profile.component.html

```diff
@@ -1,20 +1,23 @@
 <ion-header>
   <ion-toolbar>
     <ion-buttons slot="start">
-      <ion-button (click)="cancel()">Cancel</ion-button>
+      <ion-button (click)="cancel()" shape="round" type="submit" color="secondary">Cancel</ion-button>
     </ion-buttons>
     <ion-buttons slot="end">
-      <ion-button (click)="confirm()" [strong]="true">Confirm</ion-button>
+      <ion-button [disabled]="!formData.valid" (click)="confirm()" [strong]="true" shape="round" type="submit"
```

Comment:
No need for square brackets when passing a string or a boolean value. I suggest you change [stong]="true" for strong="true"